packages/perseus/src/types.ts

```diff
@@ -182,15 +195,7 @@ export type APIOptions = Readonly<{
     // A function that is called when the user has interacted with a widget. It
     // also includes any extra parameters that the originating widget provided.
     // This is used for keeping track of widget interactions.
-    trackInteraction?: (
-        args: {
-            // The widget type that this interaction originates from
-            type: string;
-            // The widget id that this interaction originates from
-            id: string;
-            correct?: boolean;
-        } & Record<string, unknown>,
```

Comment:
This approach didn't work as the flow type get ended up generating invalid Flow (which I only discovered when I pulled this into webapp).